src/components/domain/SheetPage/MemberCarryWeightTable.tsx

```diff
@@ -33,13 +36,14 @@ const MemberCarryWeightTable: React.FC<TableProps> = ({ ...props }) => {
 						<Td>
 							{getCarriedItems(member).reduce<number>(
 								(total, current) =>
-									(total += current.weight * current.quantity),
+									(total += getItemTotalWeight(current)),
```

Comment:
Need to use `big.js` to sum the totals

Reply:
Got it, I've made the adjustments. Ah, looks like that total is in the sheetPageState.ts... I'm looking into it

![sheetPageState-getColumnSums](https://user-images.githubusercontent.com/14916065/117818058-da429080-b22d-11eb-9167-5fbe882ed853.png)